app.py

```diff
@@ -34,11 +35,13 @@ def fetch_net_ids() -> Dict[str, str]:
     response = requests.post(REDCAP_API_URL, data=data)
     response.raise_for_status()
 
+    if len(response.json()) == 0:
+        return None
+
+    user_data = response.json()[0]
```

Comment:
Do we want to assert that the net_id only matches to one record_id? 

I know in REDCap we will be marking the net_id as a unique identifier field so there should never be duplicate net_id, but should we check in case REDCap has a bug again?

Reply:
Yep, that's a great call.

Reply:
Currently, throwing an Exception creates a 500 (Internal Server Error) on the front-end. What's our desirable behavior for catching/throwing Exceptions here?

Reply:
Oh right, the participant interacts with this directly. We would want some alert to us internally, but what should be relayed to the participant? 🤔

Reply:
I'm not sure what's the best approach for handling this, but I added an `assert` in a new commit that there is only one record.

Reply:
We could return a very plain message requesting the participant contact Husky Testing support.

Reply:
I added a try/except block around fetching REDCap data in a new commit.